src/components/Navbar/DropDownMenu.js

```diff
@@ -0,0 +1,30 @@
+// Dependencies
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+
+//Components, assets, actions, styles etc..
+import { setActiveMenu } from '../../actions/index';
+
+const DropDownMenu = (props) => {
+  const dispatch = useDispatch();
+
+  const closeMenu = (e) => {
+    e.stopPropagation();
+    dispatch(setActiveMenu(false));
+  };
```

Comment:
Whats the purpose of this? I don't think you logic here works correctly

Reply:
Cleaned up!